autoload/ale/code_action.vim

```diff
@@ -63,29 +62,29 @@ function! s:ChangeCmp(left, right) abort
     return 0
 endfunction
 
-function! ale#code_action#ApplyChanges(filename, changes, should_save) abort
-    let l:current_buffer = bufnr('')
+function! ale#code_action#ApplyChanges(filename, changes, options) abort
+    let l:should_save = get(a:options, 'should_save')
+    let l:conn_id = get(a:options, 'conn_id')
+
+    let l:orig_buffer = bufnr('')
+
     " The buffer is used to determine the fileformat, if available.
     let l:buffer = bufnr(a:filename)
-    let l:is_current_buffer = l:buffer > 0 && l:buffer == l:current_buffer
 
-    if l:buffer > 0
-        let l:lines = getbufline(l:buffer, 1, '$')
-
-        " Add empty line if there's trailing newline, like readfile() does.
-        if getbufvar(l:buffer, '&eol')
-            let l:lines += ['']
-        endif
-    else
-        let l:lines = readfile(a:filename, 'b')
+    if l:buffer != l:orig_buffer
+        call ale#util#Execute('edit ' . a:filename)
```

Comment:
Add silent so neovim stops asking to press Enter to continue:

```
call ale#util#Execute('silent edit ' . a:filename)
```

Reply:
Done